Rocket.Chat/Views/Cells/Chat/ChatMessageCell.swift

```diff
@@ -87,6 +87,19 @@ final class ChatMessageCell: UICollectionViewCell {
         return total
     }
 
+    // MARK: Sequential
+    @IBOutlet weak var labelUsernameHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var labelDateHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var avatarContainerHeightConstraint: NSLayoutConstraint!
+
+    var sequential: Bool = false {
+        didSet {
+            avatarContainerHeightConstraint.constant = sequential ? 0 : 35
+            labelUsernameHeightConstraint.constant = sequential ? 0 : 21
+            labelDateHeightConstraint.constant = sequential ? 0 : 21
```

Comment:
What about even not adding values to avatar/timestamp/username labels? This way we can avoid some processing into the app.

Reply:
Yeah, i thought about that. I'll give it a second try, but as far as i know views with height/width == 0 are ignored by the rendering pipeline.

Reply:
Yes, they are. But we're still constructing the avatar here: https://github.com/RocketChat/Rocket.Chat.iOS/pull/618/files#diff-535132aa8b35719fb1bcbd5602ca482eR193.

Reply:
#[Prevent unnecessary computations for sequential cells](https://github.com/RocketChat/Rocket.Chat.iOS/pull/618/commits/987713cafcea9be45ee0558466ac337cb1a696f8)